modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -156,7 +161,9 @@ private Object[] _getManyToManyRelationshipObjectEntries(
 				manyToManyRelatedObjectEntries);
 		}
 		catch (PortalException portalException) {
-			portalException.printStackTrace();
+			if (_log.isWarnEnabled()) {
```

Comment:
Oh ! I answer myself now :joy: 

Maybe better if you fixup your commits together

Reply:
Honestly, I've tried and I had a lot of conflicts. As we are in a little bit hurry to send this code, I preferred to send it in an SF commit 😅 Sorry